fastapi_jsonapi/signature.py

```diff
@@ -50,26 +48,21 @@ def create_additional_query_params(schema: Optional[Type[BaseModel]]) -> tuple[l
     available_includes_names = []
 
     # TODO! ?
-    schema.update_forward_refs(**registry.schemas)
-    for name, field in (schema.__fields__ or {}).items():
+    schema.model_rebuild(_types_namespace=registry.schemas)
```

Comment:
I'am hesitant about this change, because now we get the `pydantic.errors.PydanticUndefinedAnnotation: name 'PostSchema'` is not defined which most likely means that `model_rebuild` works not as it should be or declared in the wrong place

Reply:
Is there any other way to update string type annotations to real objects? The only reason to do it is the need to work with real pydantic schemas when building APIs

Reply:
I added a function that allows you to update type annotations to real obj. So far, it seems to be handling the task well, and I've placed it in `schema_builder.py::string_to_schema`